src/AddressValidation.php

```diff
@@ -67,6 +78,14 @@ public function __construct($accessKey = null, $userId = null, $password = null,
     }
 
     /**
+     * This message set the addressValidationResponseObject
+     * @param bool $value
+     */
+    public function validateReturnAVObject($value = true)
```

Comment:
Yeah, I struggled with what to name that. How do you feel about `activateAVObjectReturnOnValidate` and `deactivateAVObjectReturnOnValidate`? I was trying to get the validate method name in there. It is just wordy though....